### ISSUE TITLE
TranslateTransform animation in Flipper.FrontContent doesn't work properly

### ISSUE THREAD

**Issue by Roberto2707** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2505]
Hello.
After version 4.3.0-ci82, in Flipper.FrontContent, TranslateTransform animation doesn't work properly:
```
<Window x:Class="WpfApp1.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008" xmlns:local="clr-namespace:WpfApp1"
        xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        Title="MainWindow"
        Width="300" Height="300"
        WindowStartupLocation="CenterScreen"
        mc:Ignorable="d">

    <Window.Resources>
        <Storyboard x:Key="LoadWindow">
            <DoubleAnimationUsingKeyFrames Storyboard.TargetName="txtUser" Storyboard.TargetProperty="(UIElement.RenderTransform).(TransformGroup.Children)[3].(TranslateTransform.X)">
                <EasingDoubleKeyFrame KeyTime="0" Value="800" />
                <EasingDoubleKeyFrame KeyTime="0:0:0.5" Value="800" />
                <EasingDoubleKeyFrame KeyTime="0:0:0.7" Value="0" />
            </DoubleAnimationUsingKeyFrames>
            <DoubleAnimationUsingKeyFrames Storyboard.TargetName="AcceptButton" Storyboard.TargetProperty="(UIElement.RenderTransform).(TransformGroup.Children)[3].(TranslateTransform.X)">
                <EasingDoubleKeyFrame KeyTime="0" Value="800" />
                <EasingDoubleKeyFrame KeyTime="0:0:0.5" Value="800" />
                <EasingDoubleKeyFrame KeyTime="0:0:0.9" Value="0" />
            </DoubleAnimationUsingKeyFrames>
            <DoubleAnimationUsingKeyFrames Storyboard.TargetName="CancelButton" Storyboard.TargetProperty="(UIElement.RenderTransform).(TransformGroup.Children)[3].(TranslateTransform.X)">
                <EasingDoubleKeyFrame KeyTime="0" Value="800" />
                <EasingDoubleKeyFrame KeyTime="0:0:0.5" Value="800" />
                <EasingDoubleKeyFrame KeyTime="0:0:1.1" Value="0" />
            </DoubleAnimationUsingKeyFrames>
        </Storyboard>
    </Window.Resources>

    <Window.Triggers>
        <EventTrigger RoutedEvent="Loaded">
            <BeginStoryboard Storyboard="{StaticResource LoadWindow}" />
        </EventTrigger>
    </Window.Triggers>

    <Grid Margin="10">
        <materialDesign:Flipper SnapsToDevicePixels="True">
            <materialDesign:Flipper.FrontContent>
                <StackPanel>
                    <TextBox x:Name="txtUser">
                        <TextBox.RenderTransform>
                            <TransformGroup>
                                <ScaleTransform />
                                <SkewTransform />
                                <RotateTransform />
                                <TranslateTransform />
                            </TransformGroup>
                        </TextBox.RenderTransform>
                    </TextBox>

                    <Button x:Name="AcceptButton"
                            Margin="0,15,0,5"
                            materialDesign:ShadowAssist.ShadowDepth="Depth2" KeyboardNavigation.IsTabStop="False">
                        <Button.RenderTransform>
                            <TransformGroup>
                                <ScaleTransform />
                                <SkewTransform />
                                <RotateTransform />
                                <TranslateTransform />
                            </TransformGroup>
                        </Button.RenderTransform>
                        <StackPanel Orientation="Horizontal">
                            <materialDesign:PackIcon Width="24" Height="24" Kind="AccountCheck" />
                            <TextBlock Margin="4,0,0,0" VerticalAlignment="Center" Text="Aceptar" />
                        </StackPanel>
                    </Button>

                    <Button x:Name="CancelButton"
                            Margin="0,5"
                            materialDesign:ShadowAssist.ShadowDepth="Depth2" KeyboardNavigation.IsTabStop="False">
                        <Button.RenderTransform>
                            <TransformGroup>
                                <ScaleTransform />
                                <SkewTransform />
                                <RotateTransform />
                                <TranslateTransform />
                            </TransformGroup>
                        </Button.RenderTransform>
                        <StackPanel Orientation="Horizontal">
                            <materialDesign:PackIcon Width="24" Height="24" Kind="CloseBox" />
                            <TextBlock Margin="4,0,0,0" VerticalAlignment="Center" Text="Cancelar" />
                        </StackPanel>
                    </Button>
                </StackPanel>
            </materialDesign:Flipper.FrontContent>
        </materialDesign:Flipper>
    </Grid>
</Window>

```
Until version 4.3.0-ci82, the controls inside Flipper.FrontContent move from right to left until they were positioned on the left side, pretending to appear when loading the window. Now, they appear and are resized.

**Comment by Roberto2707** [comment]
Is there any way to download version 4.3.0-ci82 and next version 4.3.0-ci89 to check which is the problem?
I don't know if this is posible. In the new version 4.5.0 it is still not fixed. Thank you.

**Comment by Keboo** [comment]
Sorry I had intended to bump out all of the open items from 4.5.0 to 4.6.0 after the release last night and forgot.

Yes, all version are available on NuGet.org.
https://www.nuget.org/packages/MaterialDesignThemes/
The simplest option is within Visual Studio you can open your NuGet package manager, and simply set it to show Preview NuGet packages. From there you can use the version drop down to select whichever versions you want.


Alternatively, under the Versions tab in NuGet.org you can pick specific versions.
This will let you download any specific version you want. You then need a local NuGet folder from which to install the NuGet from:
https://docs.microsoft.com/en-us/nuget/hosting-packages/local-feeds

**Comment by Roberto2707** [comment]
Hello. Thanks for answering. I already knew how to select different versions with nuget. This is what I'm doing to check if the problem is solved. What I was referring to is to download the MaterialDesignInXaml development to help find what had changed to try to fix it. Thanks again.

**Comment by Roberto2707** [comment]
I think the problem is in the ViewPort3D that is used in Plane3D. In some version change in the PresentationFramework.dll assembly, in the System.Windows.Controls namespace.

**Comment by Roberto2707** [comment]
Fixed in version 4.8.1-ci192

**Comment by Roberto2707** [comment]
Hello. After version 5.01-ci598 it has malfunctioned again.
